.vscode/cSpell.json

```diff
@@ -1,121 +1,315 @@
 {
   "version": "0.2",
   "language": "en",
+  "ignorePaths": [
+    "**/pnpm-lock.yaml"
+  ],
   "words": [
+    "aabbcc",
+    "aabbccff",
+    "abcbd",
+    "accu",
     "accudraw",
+    "accudrawcompassmodechanged",
+    "accudrawrotationchanged",
+    "activecontentchanged",
+    "activehighlight",
     "activehover",
-    "addon",
+    "activeviewportchanged",
+    "adaptee",
+    "addbuildtag",
+    "addsubunit",
+    "älles",
+    "alphasomething",
+    "andalso",
     "appui",
+    "appuiprovider",
+    "arrowdown",
+    "arrowleft",
+    "arrowup",
+    "asdtestasd",
+    "ator",
+    "autodocs",
     "autoflip",
-    "backend",
+    "autoinstaller",
+    "autoinstallers",
+    "baaab",
+    "backgr",
+    "backstageevent",
+    "badeditor",
+    "badtype",
     "betools",
+    "borderless",
     "boxshadow",
     "buic",
     "buttongroup",
-    "CADID",
-    "certa",
-    "changelog",
-    "changelogs",
-    "CODEOWNERS",
-    "coff",
+    "cadid",
+    "capturable",
+    "changehistory",
+    "checklistbox",
+    "checkmark",
+    "childwindow",
+    "codemod",
+    "codemods",
+    "codeowners",
     "colorpicker",
-    "coord",
-    "d",
-    "dataset",
-    "deserialization",
-    "deserialize",
-    "devtool",
-    "devtools",
+    "colorthemes",
+    "colorvariables",
+    "configurableui",
+    "contentcontrolactivated",
+    "contentlayout",
+    "contentlayoutactivated",
+    "csvg",
+    "cursormenu",
+    "cursorpopup",
+    "cursorprompt",
+    "customnumber",
+    "cves",
+    "dataproviders",
+    "datasource",
+    "datepicker",
+    "dblspy",
+    "decrementor",
+    "descr",
+    "deselections",
+    "distanceangle",
+    "dockings",
+    "dont",
     "downvote",
+    "dragdrop",
+    "ecef",
     "ecschema",
     "ecsql",
-    "focusable",
-    "frontend",
+    "élêçtèd",
```

Comment:
Should we simply disable checking for some of these? I.e. things like these translation strings, hex values or other one-offs? `// cspell:disable-next-line`

Reply:
I think so. At least for the v1 of the pr removed all the `// cspell:disable-next-line` as the item requested, but it does seem as a way better approach in some places.

Reply:
Removed not real words from `cSpell.json` and used `// cspell:disable-next-line` instead